monitoring/system_pods.go

```diff
@@ -29,13 +29,10 @@ import (
 	log "github.com/sirupsen/logrus"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/fields"
 )
 
 // SystemPodsConfig specifies configuration for a system pods checker.
 type SystemPodsConfig struct {
-	// NodeName specifies the kubernetes name of this node.
-	NodeName string
```

Comment:
Hmm, we're removing the `nodeName` field selector so that master nodes will also check the status of pods running on non-master nodes? That makes sense.
One concern is we will end up with duplicate probes on all master nodes with this change.

Reply:
Correct, we don't currently have a way to deduplicate checks. Also, it's sort of the case for other checks like nethealth or time sync. All masters will report that. 

Yes it should be better, but it's within the realm of the current design.

Reply:
Ideally we'd also be able to have a master node return a result for a different node, so the status would ultimately be displayed under say a worker node. I might add that later, but for this change I need to work with what we have.